ocaml/xenopsd/test/dune

```diff
@@ -16,23 +16,14 @@
   xapi-log
   xapi-stdext-pervasives
   xapi-test-utils
-  xapi-xenopsd
+  xapi_xenopsd
   xenstore_transport.unix
  )
  (preprocess
   (per_module ((pps ppx_deriving_rpc) Test))
  )
 )
 
-(rule
-  (alias runtest)
-  (package xapi-xenopsd)
```

Comment:
Can't this test be run as part of another package?

Reply:
The .cmxs file is not being built anymore, because the xenopsd library is now an internal one
(the .cmx is, but we can't use `ldd` on it to check linkage, we'd have to use `ocamlobjinfo`).

We still have this test:
```
ocaml/xenopsd/test/dune:  (action (run ./check-no-xenctrl.sh %{x}))
```

Although it is a bit odd that we don't have any checks that *xapi* doesn't link xenctrl, that was the main goal after all...
I've added a test for that here to replace the cmxs test:
2f071feec603 ("CP-51479: add test that XAPI does not depend on libxenctrl")

Reply:
We do have the check on xapi, it's in ocaml/tests/dune